src/monitor.c

```diff
@@ -458,3 +470,59 @@ struct wlr_box monitor_get_active_geom(struct monitor *m)
     struct wlr_box geom = tag_get_active_geom(tag);
     return geom;
 }
+
+void handle_output_mgr_apply(struct wl_listener *listener, void *data)
+{
+	struct wlr_output_configuration_v1 *config = data;
+	handle_output_mgr_apply_test(config, false);
+}
+
+// apply_output_config
+void handle_output_mgr_apply_test(struct wlr_output_configuration_v1 *config, bool test)
+{
+	struct wlr_output_configuration_head_v1 *config_head;
+	bool ok = true;
+
+	wl_list_for_each(config_head, &config->heads, link) {
+		struct wlr_output *wlr_output = config_head->state.output;
+
+		wlr_output_enable(wlr_output, config_head->state.enabled);
+
+		if (config_head->state.enabled) {
```

Comment:
Oh and to make things more understandable use for each if statement here a static function with the name explaining what is being done. And to not go over 80 spaces that much :)

Reply:
static functions in c are just local functions not the same as in languages such as java or lua

Reply:
Im confused about what you mean mith 80 spaces and do you mean that the functions should be static?

Reply:
you have to convert the body of the if statements to own functions that you call. Those functions should be static

Reply:
80 spaces means that a line should at most be 80 spaces long. Is that the case here though?

Reply:
I don't have a texteditor for github setup that shows me that so it was only gut feeling.

Reply:
static function in C description from google: A static function in C is a function that has a scope that is limited to its object file. This means that the static function is only visible in its object file. A function can be declared as static function by placing the static keyword before the function name.